apps/example-orchestrator-ui/components/StartPage/StatCards.tsx

```diff
@@ -0,0 +1,76 @@
+import React, { ReactElement, useState } from 'react';
+import {
+    EuiAvatar,
+    EuiFlexGroup,
+    EuiFlexItem,
+    EuiPanel,
+    EuiText,
+} from '@elastic/eui';
+
+export interface TotalStat {
+    icon: string;
+    name: string;
+    value: number;
+    color: string;
+}
+
+const totalStats = [
```

Comment:
Colors should be picked from the themeProvider. The useOrchestratorTheme hook gives the theme and some helper functions.